packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -32,7 +32,8 @@
     "@material-ui/lab": "4.0.0-alpha.45",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-use": "^17.2.4"
+    "react-use": "^17.2.4",
+    "@types/react": "^16.9"
```

Comment:
This should be a devDependency.

I've been looking through a couple of the existing plugins in the repo and this dep is often not there. When is it truly needed and when isn't it?

Reply:
For the discussion about regular / dev dependency, please see the thread created by @benjdlambert above.
However, you may be right. I guess it would only be needed when using some of the React types. It's surprising to me that this isn't universally the case among the frontend. If it isn't universally needed this PR should be closed.